src/test/java/org/jitsi/meet/test/util/MeetUtils.java

```diff
@@ -123,6 +123,22 @@ public static String getResourceJid(WebDriver participant)
             .executeScript("return APP.conference.getMyUserId();");
     }
 
+    /**
+     * Returns the value for the given <tt>key</tt> from the config.js loaded
+     * for the given <tt>participant</tt>.
+     *
+     * @param participant the <tt>WebDriver</tt> instance which runs conference
+     *                    participant.
+     * @param key the <tt>String</tt> key from config.js.
+     * @return the value for the given <tt>key</tt> from the config.js loaded
+     * for the given <tt>participant</tt>.
+     */
+    public static Object getConfigValue(WebDriver participant, String key)
+    {
+        return ((JavascriptExecutor) participant)
+                .executeScript("return config." + key);
```

Comment:
I don't like this approach for getting config parameters, because it requires alive and connected to an existing conference WebParticipant. I don't see any of such assumptions being mentioned anywhere around here.

Reply:
OK. Could you be more specific which approach you would like for this situation? If you don't have better approach in mind, the current approach will be still the best....

Reply:
Probably at least moving this method to WebParticipant, because having it there means that it belongs to web only.

Reply:
OK